reader/components/enhancement-sidebar/EnhancementSidebar.jsx

```diff
@@ -1,18 +1,36 @@
 import React from 'react'
-import { sortBy } from 'lodash'
 import './EnhancementSidebar.scss'
 import { useGlobalStore } from '../../store'
 
 const EnhancementSidebar = () => {
   const [{ ui, document }] = useGlobalStore()
 
-  const sortedAnnotations = sortBy(document.annotations, [
-    annotation => sortBy(Object.keys(annotation.rects))[0],
-    annotation =>
-      annotation.rects[sortBy(Object.keys(annotation.rects))[0]][0].top,
-    annotation =>
-      annotation.rects[sortBy(Object.keys(annotation.rects))[0]][0].left,
-  ])
+  const sortedAnnotations = Object.keys(document.annotations)
+    .map(annotationId => [annotationId, document.annotations[annotationId]])
```

Comment:
There is `Object.entries()` that already does what is written in these 2 lines.

Reply:
https://github.com/oacore/reader/pull/104/commits/d4b1c329fc788693ce580fa9b7ec9cfe759d33a6